transform-plugins/src/main/java/co/cask/hydrator/plugin/CSVParser.java

```diff
@@ -272,27 +275,31 @@ private StructuredRecord createStructuredRecord(@Nullable CSVRecord record, Stru
     @Description("Specifies the schema that has to be output.")
     private final String schema;
 
-    public Config(String format, Character delimiter, String field, String schema) {
-      this.format = format;
+    public Config(@Nullable String format, @Nullable Character delimiter, String field, String schema) {
+      this.format = format == null ? "DEFAULT" : format;
       this.delimiter = delimiter;
       this.field = field;
       this.schema = schema;
     }
 
     private void validate() {
-      // Check if the format specified is valid.
-      if (format == null || format.isEmpty()) {
-        throw new IllegalArgumentException("Format is not specified. Allowed values are DEFAULT, EXCEL, MYSQL," +
-                                             " RFC4180, Pipe Delimited, Tab Delimited or Custom");
+
+      if (format == null) {
```

Comment:
ok this is duplicating a check that happens later so you don't have to worry about null pointers... Just set the format to "default" in a no-argument constructor and you don't have to worry about it ever being null and don't need this part at all.